k8s/showcase/vineyard-mars-pytorch/prepare-data/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM ghcr.io/v6d-io/v6d/vineyard-python-dev
+
+# Install prerequisites
+RUN apt-get update && apt-get install -y \
+    curl --no-install-recommends
+
+RUN curl -LO https://dl.k8s.io/release/v1.24.0/bin/linux/amd64/kubectl && \
+    chmod +x kubectl && \
+    mv kubectl /usr/bin && \
+    pip3 install --no-cache-dir kubernetes
+
+WORKDIR /
+
+# the datasets can be found in the git repo(https://github.com/GraphScope/gstest/tree/master/vineyard-mars-showcase-dataset)
+COPY datasets/ datasets/
+
+COPY prepare-data.py /
+
+ENTRYPOINT ["python3", "prepare-data.py"]
```

Comment:
Looks same with `k8s/test/e2e/Dockerfile`, except the dataset.

Don't repeat yourself and avoid coping similar files again and again. Mounting the dataset from hosts into the kind workers and then use `hostPath` or `local-path-provider` volumes would better and more scalable in the future.

Reply:
Make sense.